### ISSUE TITLE
Out of the box `init --template rebolt` project doesn't work

### ISSUE THREAD

**Issue by FLGMwt** [callstackincubator/rebolt#41]
The [suggested command](https://rebolt.callstack.com/docs/get-started.html#starting-a-fresh-react-native-project):

`react-native init Foo --template rebolt`

results in a project that isn't clear how to run. Without any npm scripts included in the template, my assumptive `react-native run-android` and `react-native run-ios` both result in builds that start Metro (not haul) and complain about not being able to find `src/App.bs.js`.

Including screenshots because neither log-ios nor log-android captured this output: 

![image](https://user-images.githubusercontent.com/4138357/42351216-783faee2-8079-11e8-985b-f56d7ab353f6.png)

![image](https://user-images.githubusercontent.com/4138357/42351219-7d176f9a-8079-11e8-9835-d3e333d457a5.png)

# Environment 

OSX: 10.13.3
yarn: 1.7.0
react-native-cli: 2.0.1
react-native: 0.56.0

# Solution(?)

If there are manual steps required such as those enumerated in the [steps to add rebolt to an existing project](https://rebolt.callstack.com/docs/get-started.html#adding-reasonml-to-your-existing-react-native-project), that should be either enumerated in the template docs or included in the template.

If I have some time, I'll look into the minimal overlap of template/manual work.

**Comment by knowbody** [comment]
It's a weird issue, I did the `init` myself and got the same error. But after I did a `bsb -clean-world` and rebuild it worked. and I couldn't reproduce it.

I tried to `init` once again, and I ran `yarn watch` before I run the `react-native run-ios` and it worked straight away.

These are the missing scripts in `package.json` that are used to build, watch and clean:

```json
"build": "bsb -make-world",
"start": "bsb -make-world -w",
"clean": "bsb -clean-world",
```

Could you try it once again and let me know?

**Comment by knowbody** [comment]
@FLGMwt let me know if this is still happening, otherwise we will close the issue as I cannot reproduce it anymore

**Comment by FLGMwt** [comment]
Yup, can confirm everything is working once I add the bsb npm scripts.

Given "This will create a fresh React Native project using rebolt template, giving you the already preconfigured Reason environment out of the box.", I was expecting to be able to run the app without changes. Given two options of "Starting a fresh React Native project" and "Adding ReasonML to your existing React Native project", only the latter has the npm script instructions.

It looks like the current RN templating support doesn't allow for adding template-specific npm scripts, so I think we should update the docs to reference the required scripts.

**Comment by knowbody** [comment]
@FLGMwt would you mind opening the PR with what would be a better way to communicate it?

Like you said the RN templating doesn't allow us to do everything, but @grabbou included READMEs in the project that gets initialized, but maybe we should also have a clearer doc in the getting started.

**Comment by knowbody** [comment]
I added an issue for it so we won't forget about fixing it: https://github.com/callstackincubator/rebolt/issues/53
